momaland/utils/aec_wrappers.py

```diff
@@ -0,0 +1,130 @@
+"""Various wrappers for AEC MO environments."""
+
+import numpy as np
+
+
+class Wrapper:
+    """Base class for wrappers."""
+
+    def __init__(self, env):
+        """Base wrapper initialization to save the base env."""
+        self._env = env
+
+    def __getattr__(self, name):
+        """Provide proxy access to regular attributes of wrapped objects."""
+        return getattr(self._env, name)
+
+
+class LinearizeReward(Wrapper):
+    """Convert MO reward vector into scalar SO reward value.
+
+    `weights` represents the weights of each objective in the reward vector space.
+    """
+
+    def __init__(self, env, weights: np.ndarray):
```

Comment:
I believe this makes more sense to have a dict or weights, to have the possibility to specify weights for each agent?